adjustPipelines.sh

```diff
@@ -85,8 +84,8 @@ do
 					logicalname=$(name $repo)
 					if changeDir $repository
 					then
-						sed_inplace "s?http://services.typefox.io/open-source/jenkins/job/$logicalname/job/[^/]*/lastStableBuild/artifact/build/p2-repository/?http://services.typefox.io/open-source/jenkins/job/$logicalname/job/master/lastStableBuild/artifact/build/p2-repository/?" $targets/*.target
-						isMaster ||	( isBranch && sed_inplace "s?http://services.typefox.io/open-source/jenkins/job/$logicalname/job/master/lastStableBuild/artifact/build/p2-repository/?http://services.typefox.io/open-source/jenkins/job/$logicalname/job/$escapedBranch/lastStableBuild/artifact/build/p2-repository/?" $targets/*.target )
+						sed_inplace "s?$JENKINS_URL/job/$logicalname/job/[^/]*/lastStableBuild/artifact/build/p2-repository/?$JENKINS_URL/job/$logicalname/job/master/lastStableBuild/artifact/build/p2-repository/?" $targets/*.target
```

Comment:
Isn't it the case that a $ is a special character in sed regexes?

Reply:
Actually the `$JENKINS_URL` is replaced by the variable value before it calls sed. So sed only gets the real value.

I have updated the branch to use a default for JENKINS_URL and use the environment variable when available. This will also work when the JENKINS_URL changes in the target files.

Resulting output is
```
export JENKINS_URL=https://ci.eclipse.org/xtext
./adjustPipelines.sh kth_issue1224

JENKINS_URL=https://ci.eclipse.org/xtext
Redirecting target platforms in /Users/thoms/Development/Eclipse/xtext-dev/git/xtext-umbrella/releng/org.eclipse.xtext.sdk.target to kth_issue1224
```

Diff:
![screenshot 15](https://user-images.githubusercontent.com/265597/45291558-d1381900-b4f2-11e8-8cec-6aeffc65a2e9.png)